docs/docs-components/COMPONENT_DATA.js

```diff
@@ -2562,6 +2562,56 @@ const GENERAL_COMPONENT_LIST: $ReadOnlyArray<ListItemType> = [
       figma: null,
     },
   },
+  {
+    svg: <Table />,
+    name: 'TableOfContents',
+    aliases: ['Table of Contents', 'Page Navigation', 'Page Navigator', 'Secondary Nav'],
+    description:
+      'TableOfContents component is used to navigate to anchors on a page. It also serves as an outline of a page’s content. TableOfContents is placed on the right side of the page, close to the main content block.',
+    category: 'Navigation',
+    status: {
+      accessible: {
+        summary: 'ready',
+        a11yVisual: 'ready',
+        a11yScreenreader: 'ready',
+        a11yNavigation: 'ready',
+        a11yComprehension: 'ready',
+      },
+      badge: null,
+      android: 'notAvailable',
+      deprecated: false,
+      documentation: 'ready',
+      iOS: 'notAvailable',
+      figma: 'ready',
+      responsive: 'ready',
```

Comment:
@AlbertCarreras I am not sure about this property. ToC is not designed for mobile and it should disappear on mobile screens. What should I set here?

Reply:
I think it's if a mobile consideration has been made, I think marking as notReady is okay, if it completely disappears?